Containerfile.task

```diff
@@ -4,7 +4,15 @@
 # in-cluseter with the tasks focused on building remotely. Therefore, we need to maintain a buildah image
 # for our tasks that has more than _just_ buildah in it. We also need to add the required functionality
 # for the remote builds.
+
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21 AS builder
+
+WORKDIR /app
+RUN git clone https://github.com/mkosiarc/dockerfile-json.git .
```

Comment:
Are we going to move this to a centrally managed repository or just leave it here?

Another option is to add the original source as a submodule and then store a patch which we can apply at build time to fix this simple change.

Reply:
Moved the repo to konflux-ci